src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,85 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesCount = getNumbers(1, Math.ceil(total / perPage));
+
+  return (
+    <ul className="pagination">
+      <li className={classNames(
+        'page-item',
+        {
+          disabled: currentPage === 1,
+        },
+      )}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => {
+            if (currentPage !== 1) {
```

Comment:
It can be used here too

Reply:
sorry, i cant find where they repeated :(